.template/spec/addons/base/github/template_spec.rb

```diff
@@ -21,9 +21,7 @@
 
   # TODO: Can't test this as it is now ignored by `.dockerignore`
   xit 'modifies the README.md' do
-    expect(file('README.md')).to contain('## Documentation')
-
-    expect(file('README.md')).not_to contain('## Getting Started')
+    expect(file('README.md')).not_to contain('## Get Started')
```

Comment:
Is this change needed?  🤔

Reply:
Thanks 👍🏼 . Reverted in c98430ab389ec98d650d02182b2fd473096f7529